custodia/store/sqlite.py

```diff
@@ -110,8 +110,11 @@ def list(self, keyfilter=''):
                     continue
                 if not row[0].startswith(child_prefix):
                     continue
-                value.append(row[0][len(child_prefix):].lstrip('/'))
-
+                result = row[0][len(child_prefix):].lstrip('/')
+                if not self.get(row[0]):
```

Comment:
We have 2 options here.
1. SELECT key, value FROM ...
If we do not want to fetch all values, then we would have to add a column with the type.
I always wanted to add a type because it will allow us later to define (optionally) some behavior, but that requires some architecting to avoid creating a system where it will not be possible to upgrade or deal with external storage that can't easily represent that value.
So I think you can get the value internally for now and open a PR to add row types in the sqlite database.